actix-web-flash-messages/src/storage/cookies.rs

```diff
@@ -77,8 +83,8 @@ impl CookieMessageStore {
                 .secure(true)
                 .http_only(true)
                 .same_site(SameSite::Lax)
-                // In the future, consider making the `path` configurable - either globally or on a per-endpoint basis
-                .path("/")
+                .path(&self.path)
+                .domain(&self.domain)
```

Comment:
I'm not sure setting an empty domain is equivalent to skipping the call to `domain`.

Reply:
It isn't:

> The Domain attribute specifies which [host](https://developer.mozilla.org/en-US/docs/Glossary/Host)s can receive a cookie. If unspecified, the attribute defaults to the same host that set the cookie, excluding subdomains. If Domain is specified, then subdomains are always included. Therefore, specifying Domain is less restrictive than omitting it. However, it can be helpful when subdomains need to share information about a user.

From https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies

We need to leave it unset if it isn't specified by the user.